File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        freq = [0] * 3
        for i in nums:
            freq[i] += 1
        
        count = 0
        for i in range(3):
            nums[count: count + freq[i]] = [i] * freq[i]
            count += freq[i]
        

File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int freq[3] = {0};
        for (int i : nums){
            freq[i]++;
        }
        int n = nums.size(), count = 0;
        for (int i = 0; i < 3; i++){
            fill(nums.begin() + count, nums.begin() + count + freq[i], i);
            count += freq[i];
        }
    }
};